TomTomSDKExamples/Examples/UseCase/BasicNavigationMenu.swift

```diff
@@ -0,0 +1,172 @@
+
+                        Text(Resource.Strings.online)
+                            .foregroundColor(Resource.Colors.bodyLight)
+                    }
+                    .onTapGesture {
+                        selectedMapMode = .online
+                    }
+
+                    // Hybrid
+                    VStack(alignment: .center, spacing: 16) {
+                        Image(Resource.Images.hybrid)
+                            .modifier(MenuModifier(isSelected: selectedMapMode == .hybrid))
+
+                        Text(Resource.Strings.hybrid)
+                            .foregroundColor(Resource.Colors.bodyLight)
+                    }
+                    .onTapGesture {
+                        selectedMapMode = .hybrid
+                    }
+
+                    // Offline
+                    VStack(alignment: .center, spacing: 16) {
+                        Image(Resource.Images.offline)
+                            .modifier(MenuModifier(isSelected: selectedMapMode == .offline))
+
+                        Text(Resource.Strings.offline)
+                            .foregroundColor(Resource.Colors.bodyLight)
+                    }
+                    .onTapGesture {
+                        selectedMapMode = .offline
+                    }
+                }
+
+                Spacer()
+
+                Button {
+                    // Open the navigation app
+                    showSheet = true
+                } label: {
+                    Text(Resource.Strings.next)
+                        .font(.body)
+                        .fontWeight(.bold)
+                        .foregroundColor(Resource.Colors.primaryContent)
+                        .frame(minWidth: 0, maxWidth: .infinity)
+                        .padding()
+                        .background(
+                            Capsule()
+                                .fill(selectedMapMode == .none ? Resource.Colors.inactiveLight : Resource.Colors.primaryLight)
+                                .shadow(color: Resource.Colors.boxShadow, radius: 8, x: 0, y: 0)
+                        )
+                }
+                .disabled(selectedMapMode == nil)
+                .fullScreenCover(isPresented: $showSheet) {
+                    navigationContent
+                }
+            }
+            .padding()
+
+            // MARK: Back Button
+
+            BackButton(useShadow: false, presentationMode: presentationMode)
+                .padding(.vertical)
+        })
+        .navigationBarHidden(true)
+    }
+
+    // MARK: - Internal
+
+    enum MapMode {
+        case online
+        case hybrid
+        case offline
+    }
+
+    @Environment(\.presentationMode)
+    var presentationMode
+
+    @State
+    var selectedMapMode: MapMode?
+
+    @State
+    var showSheet: Bool = false
+
+    @ViewBuilder
+    var navigationContent: some View {
+        switch selectedMapMode {
+        case .online:
+            BasicNavigationView()
```

Comment:
For symmetry I'd suggest using `OnlineNavigationView` instead of `BasicNavigationView`, so it would be easier to find it and understand what's inside.

Reply:
Done!